src/libs/TransactionUtils/index.ts

```diff
@@ -832,6 +836,14 @@ function isPosted(transaction: Transaction): boolean {
     return transaction.status === CONST.TRANSACTION.STATUS.POSTED;
 }
 
+/**
+ * The transaction is considered scanning if it is a partial transaction, has a receipt, and the receipt is being scanned.
+ * Note that this does not include receipts that are being scanned in the background for auditing / smart scan everything, because there should be no indication to the user that the receipt is being scanned.
+ */
+function isScanning(transaction: OnyxEntry<Transaction>): boolean {
```

Comment:
NAB - this might end up being a little confusing for internal engineers because we have another `isScanning` function in Web-E that's only based on the receipt state (more like `isReceiptBeingScanned()` here).

Based on the description and how it's used, I think this makes sense here though so NAB.